nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -1320,6 +1317,7 @@ Make sure to also check the many updates in the [Nixpkgs library](#sec-release-2
   The configuration file could previously be read, when `userControlled.enable` (non-default),
   by users who are in both `wheel` and `userControlled.group` (defaults to `wheel`)
 
+- The `drbd` out-of-tree Linux kernel driver has been added in version `9.2.7`. With it the DRBD 9.x features can be used instead of the 8.x features provided by the `8.4.11` in-tree driver.
```

Comment:
please drop this change since 23.11 was already released.

Reply:
done